run_tests.py

```diff
@@ -0,0 +1,141 @@
+'''
+This script sets up role assumption for the MongoDB Module
```

Comment:
Can you elaborate on when/why/how a user should use this file. I think its important to note that this is used for local testing and is only needed when you need to run tests while assuming a role like the mongodb module.

Also could we put this in the test/ directory?